File: petshop-be/service/src/main/java/com/turkninja/petshop/question/QuestionService.java
package com.turkninja.petshop.question;

import com.turkninja.petshop.api.request.question.CreateQuestionRequest;
import com.turkninja.petshop.api.response.common.PageResponse;
import com.turkninja.petshop.api.response.question.CreateQuestionResponse;
import com.turkninja.petshop.api.response.question.GetQuestionResponse;

public interface QuestionService {
     CreateQuestionResponse createQuestion(CreateQuestionRequest request);
     GetQuestionResponse GetQuestionById(long questionId);
     PageResponse<GetQuestionResponse> getAllQuestions(int page,int size);
}

File: petshop-admin-panel/src/containers/question/QuestionService.js
import request from "../../common/request";

export const getQuestionById = (questionId)=>{

    const questionRequest= {
        method: "GET",
        path: "/api/v1/question/get-question-by-id",
        params: ["questionId"],
        values: [questionId]
    }
    return request(questionRequest)

}

export const getAllQuestions = (page,size) => {

    const questionGetAllQuestion = {
        method: "GET",
        path: "/api/v1/question/get-all-questions",
        params: ["page","size"],
        values: [page,size]

    }
    return request(questionGetAllQuestion);

}